.npmignore

```diff
@@ -0,0 +1,17 @@
+#tests
+tests
+coverage
+
+#build tools
+.travis.yml
+.jenkins.yml
+.codeclimate.yml
+
+#linters
+.jscsrc
+.jshintrc
+.eslintrc*
+
+#editor settings
+.idea
+.editorconfig
```

Comment:
I guess this is nice, but for the build, I included the files attribute which specifies  which files are used for publishing the packge

Reply:
I think it is nice for less size of the package when installing it

Reply:
Run `npm publish --dry-run` you on the development branch you will see what I mean, npm includes the needed files only already